tests/integration/test_enrollment/test_agent_auth_enrollment.py

```diff
@@ -0,0 +1,76 @@
+'''
+brief: This module verifies the correct behavior of the agent-auth enrollment tool under different configurations
+copyright:
+    Copyright (C) 2015-2021, Wazuh Inc.
+    Created by Wazuh, Inc. <info@wazuh.com>.
+    This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
```

Comment:
```suggestion
Copyright (C) 2015-2021, Wazuh Inc.
Created by Wazuh, Inc. <info@wazuh.com>.
This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
```

The copyright should be placed at the beginning of the file

Reply:
This is part of the DocGenerator standard:https://github.com/wazuh/wazuh-qa/issues/1626